### ISSUE TITLE
issue installing gdal python3 module 2.3.0

### ISSUE THREAD

**Issue by thehesiod** [OSGeo/gdal#674]
## Expected behavior and actual behavior.

Should be able to brew install gdal, followed by python -m pip install gdal with same version

## Steps to reproduce the problem.

- brew install gdal (verify 2.3.0 gets installed)
-  python3 -m pip install gdal==2.3.0

installing via pip yields:
```
    extensions/gdal_wrap.cpp:3173:10: fatal error: 'cpl_vsi_error.h' file not found
    #include "cpl_vsi_error.h"
             ^~~~~~~~~~~~~~~~~
    1 warning and 1 error generated.
    error: command 'clang' failed with exit status 1
```

This file does not exist in the brew casket folder or in the python module source.

## Operating system

OSX 10.13.5

## GDAL version and provenance

2.3.0 of gdal-c + 2.3.0 of gdal python module

**Comment by thehesiod** [comment]
workaround is running python with `PYTHONPATH=/usr/local/Cellar/gdal/2.3.0/lib/python3.6/site-packages`

**Comment by rouault** [comment]
Yes, known issue. Was fixed post 2.3.0 per a04416c17c264bdd8079ba7dd321fc3744175ce9 . Queued for 2.3.1
